uamqp/authentication.py

```diff
@@ -43,19 +45,22 @@ def _generate_sas_token(auth_uri, sas_name, sas_key, expiry_in=AUTH_DEFAULT_EXPI
 
 
 class SASLPlainAuth(object):
+    _auth_type = AUTH_TYPE_SASL_PLAIN
```

Comment:
I think we could just make this public. Means we don't have to suppress pylint every time we check it.